R/quick.R

```diff
@@ -0,0 +1,95 @@
+#' Quickly View Given Structure
+#'
+#' Creates a scene with a number of simple defaults in order to quickly view
+#' the structure without having to write multiple lines of code.
+#'
+#' @param model Model to add to scene. Can be \code{{bio3d}} pdb object or PDB
+#' id code string (i.e "4ozs").
+#' @param highlight Given selection will additionally have 'ball-n-stick'
+#' representation. View will also zoom to selection.
+#' @param zoom Logical. FALSE will not zoom onto highlighted selection.
+#' @param spin TRUE / FALSE will enable or disable spin. A numeric value will
+#' change spin speed and negative will reverse the direction.
+#' @param nomouse Logical. Enables / disables mouse input.
+#' @param ribbon Logical. Enables / disables ribbon representation.
+#' @param outline Logical. Enables / disables black outline.
+#' @param backgroundColor String of simple colour names or hex code to change
+#' background color of viewer.
+#'
+#' @examples
+#' library(r3dmol)
+#'
+#' # write/read demo structure as {bio3d} object
+#' tmp <- tempfile()
+#' write(pdb_6zsl, tmp)
+#' pdb <- bio3d::read.pdb(tmp)
+#'
+#' # quickly preview structure
+#' pdb %>%
+#'   m_quick_look()
+#'
+#' # preview structure, highlighting particular region.
+#' pdb %>%
+#'   m_quick_look(m_sel(resi = 1:10, chain = "A"), spin = 0.2)
+#' \dontrun{
+#' # Fetch given PDB string and quickly preview structure
+#' "4ozs" %>%
+#'   m_quick_look(spin = TRUE)
+#' }
+#' @export
+m_quick_look <- function(model,
+                         highlight = m_sel(),
+                         zoom = TRUE,
+                         spin = FALSE,
+                         nomouse = FALSE,
+                         ribbon = FALSE,
+                         outline = TRUE,
+                         backgroundColor = "white") {
+  if (is.character(model)) {
+    model <- m_fetch_pdb(model)
+  }
+
+  if (methods::is(model) == "pdb") {
```

Comment:
It seems that using the code from demo
```r
"4ozs" %>%
  m_quick_look(spin = TRUE)
```
will show a warning.

Sorry for my system language is default to Japanese.
```
警告メッセージ: 
 if (methods::is(model) == "pdb") { で: 
   条件が長さが 2 以上なので、最初の 1 つだけが使われます 
```

Reply:
Ah woops, have fixed!